pinot-common/src/main/java/org/apache/pinot/common/utils/PinotDataType.java

```diff
@@ -375,6 +530,16 @@ public double toDouble(Object value) {
       throw new UnsupportedOperationException("Cannot convert value from BYTES to DOUBLE");
     }
 
+    @Override
+    public boolean toBoolean(Object value) {
+      throw new UnsupportedOperationException("Cannot convert value from BYTES to BOOLEAN");
```

Comment:
If BYTES is not 0, it is true; otherwise, false?

Reply:
I wouldn't worry too much about that. People are not supposed to use `BYTES` as `BOOLEAN`, and throwing exception might be a safer option to avoid unexpected behavior.